include/libtorrent/error_code.hpp

```diff
@@ -326,6 +326,8 @@ namespace libtorrent {
 			// the listen socket associated with this request was closed
 			invalid_listen_socket,
 
+			// these error codes are deprecated, NAT-PMP/PCP error codes have
+			// been moved to their own category
```

Comment:
there's a `TORRENT_DEPRECATED_ENUM` macro that expands to the appropriate deprecated attribute for compilers that support it. It seems like it would make sense to mark these up, or would it trigger tons of internal warnings?